spec/system/tasks_spec.rb

```diff
@@ -0,0 +1,153 @@
+require 'rails_helper'
+
+RSpec.describe "Tasks", type: :system do
+  let(:user){ create(:user) }
+  let(:task){ create(:task) }
+
+  describe "ログイン前" do
+    describe "タスク関連ページへの遷移" do
+      context '新規登録ページへのアクセス' do
+        it '新規登録ページへのアクセス失敗' do
+          visit new_task_path
+          expect(page).to have_content('Login required')
+          expect(current_path).to eq login_path
+        end
+      end
+
+      context '編集ページへのアクセス' do
+        it '編集ページへのアクセス失敗' do
+          visit edit_task_path(task)
+          expect(page).to have_content('Login required')
+          expect(current_path).to eq login_path
+        end
+      end
+
+      context '詳細ページへのアクセス' do
+        it '詳細ページへのアクセスの成功' do
+          visit task_path(task)
+          expect(page).to have_content(task.title)
+          expect(current_path).to eq task_path(task)
+        end
+      end
+
+      context 'タスク一覧ページへのアクセス' do
+        it 'タスク一覧が表示される' do
+          task_list = create_list(:task, 3)
+          visit tasks_path
+          expect(page).to have_content task_list[0].title
+          expect(page).to have_content task_list[1].title
+          expect(page).to have_content task_list[2].title
+          expect(current_path).to eq tasks_path
+        end
+      end
+    end
+  end
+
+    describe 'ログイン後' do
+      before do
+        login_as(user)
+      end
+
+      describe 'タスクの新規作成' do
+        context "正常に入力する" do
+          it 'タスクが新規作成される' do
+            visit new_task_path
+            fill_in 'Title', with: '新規作成'
+            fill_in 'Content', with: '新規作成の内容'
+            select 'doing', from: 'Status'
```

Comment:
enumならSymbol形式で記載できる様な気がします。
```suggestion
            select :doing, from: 'Status'
```